.husky/pre-commit

```diff
@@ -2,3 +2,4 @@
 . "$(dirname "$0")/_/husky.sh"
 
 yarn lint
+yarn test
```

Comment:
no newline at the end of the file.
resolve this

Reply:
@ericus123 how can I resolve this I added test on pre commit hook.